templates/partials/person_table.html

```diff
@@ -7,25 +7,26 @@
       {% if not entity.is_department and department_salaries %}
       <th scope="col">Department</th>
       {% endif %}
-      <th scope="col" class="text-right">Salary</th>
+      <th scope="col" class="text-right">Base Salary</th>
+      <th scope="col" class="text-right">Extra Pay</th>
       {% if show_start_date %}
       <th scope="col" class="text-right">Start Date</th>
       {% endif %}
     </tr>
   </thead>
   <tbody>
-  {% for job in jobs %}
+  {% for salary in salaries %}
     <tr>
-      <td><a href="{{ url('person', kwargs={'slug': job.person.slug}) }}">{{ job.person }}</a></td>
-      <td>{{ job.position.title }}</td>
+      <td><a href="{{ url('person', kwargs={'slug': salary.job.person.slug}) }}">{{ salary.job.person }}</a></td>
+      <td>{{ salary.job.position.title }}</td>
       {% if not entity.is_department and department_salaries %}
-      <td><a href="{{ url('department', kwargs={'slug': job.position.employer.slug}) }}">{{ job.position.employer }}</a></td>
+      <td><a href="{{ url('department', kwargs={'slug': salary.job.position.employer.slug}) }}">{{ salary.job.position.employer }}</a></td>
       {% endif %}
-      {% if job.salary %}
+      {% if salary.job.salary %}
```

Comment:
Since we're working from the salary instead of the job now, can rendering base pay be simplified into simply displaying `salary.amount`? (I'll be honest with you, I don't fully remember why it's written this way to begin with. 😅)